frontend/src/main/scala/bloop/Cli.scala

```diff
@@ -124,5 +124,37 @@ object Cli {
     }
   }
 
-  def run(action: Action, logger: Logger): ExitStatus = Interpreter.execute(action, logger)
+  def run(action: Action, logger: Logger): ExitStatus = {
+    import bloop.io.AbsolutePath
+    def getConfigDir(cliOptions: CliOptions): AbsolutePath = {
+      cliOptions.configDir
+        .map(AbsolutePath.apply)
+        .getOrElse(cliOptions.common.workingPath.resolve(".bloop-config"))
+    }
+
+    import bloop.engine.{State, Build}
+    def loadStateFor(configDirectory: AbsolutePath, logger: Logger): State = {
+      State.stateCache.getBuildFor(configDirectory) match {
```

Comment:
`getBuildFor` returns a `State`?

Reply:
No, it returns an option of state, why?

Reply:
I meant that the name implies that it returns an option of Build. Shouldn’t the method be renamed?